File: src/main/java/com/curseclient/mixin/accessor/entity/AccessorEntityPlayerSP.java
package com.curseclient.mixin.accessor.entity;

import net.minecraft.client.entity.EntityPlayerSP;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(EntityPlayerSP.class)
public interface AccessorEntityPlayerSP {

    @Accessor("lastReportedYaw")
    float getLastReportedYaw();

    @Accessor("lastReportedYaw")
    void setLastReportedYaw(float value);

    @Accessor("lastReportedPitch")
    float getLastReportedPitch();

    @Accessor("lastReportedPitch")
    void setLastReportedPitch(float value);
}

File: src/main/kotlin/com/curseclient/mixin/accessor/entity/AccessorEntityPlayerSP.kt
package com.curseclient.mixin.accessor.entity

import net.minecraft.client.entity.EntityPlayerSP
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(EntityPlayerSP::class)
interface AccessorEntityPlayerSP {
    @get:Accessor("lastReportedYaw")
    @set:Accessor("lastReportedYaw")
    var lastReportedYaw : Float

    @get:Accessor("lastReportedPitch")
    @set:Accessor("lastReportedPitch")
    var lastReportedPitch : Float
}